meilisearch-http/Cargo.toml

```diff
@@ -5,18 +5,28 @@ edition = "2018"
 license = "MIT"
 name = "meilisearch-http"
 version = "0.21.0-alpha.3"
+
 [[bin]]
 name = "meilisearch"
 path = "src/main.rs"
 
 [build-dependencies]
+actix-web-static-files = { git = "https://github.com/MarinPostma/actix-web-static-files.git", rev = "6db8c3e" }
```

Comment:
Isn't it an optional dep too?

Reply:
not for build? idk I I have conditional compilation enabled for build.rs, gonna check

Reply:
turns out you can! :)